Supernova.Enum.Generators/EnumGeneratorAttributeHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Supernova.Enum.Generators;
+
+public static class EnumGeneratorAttributeHelper
+{
+    public const string Name = "EnumGenerator";
+
+    public const string Content = @"// <auto-generated />
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace " + SourceGeneratorHelper.NameSpace + @"
+{
+    [AttributeUsage(AttributeTargets.Enum)]
+    [GeneratedCodeAttribute(""Supernova.Enum.Generators"", null)]
```

Comment:
Does adding Generated Code Attribute make a special change?

Reply:
It's a support attribute for tools that use Reflection for analysis. Does the same thing as the auto-generated tag for them.